.github/CODEOWNERS

```diff
@@ -2,6 +2,12 @@
 # except files which have code owners.
 *                    @pocke
 
+# Misc
```

Comment:
Move the `# Misc` section before `/gems` section to make the updating the lines simpler.